lib/ts/setup-scripts/frontend/supertokens-react-custom.ts

```diff
@@ -0,0 +1,34 @@
+import { Answers } from "../../types";
+import path from "path";
+import fs from "fs/promises";
+import { fileURLToPath, pathToFileURL } from "url";
+
+export default async function main(frontendDirectory: string, answers: Answers) {
```

Comment:
i dont see why we need scripts for setting up stuff, when we already have the cli running

Reply:
The scripts are part of the CLI itself. I just organised them in a separate folder with emphasis on the name. It is like `setup-scripts/${location.main}`, this avoids writing **if/else** logic. The scripts are not part of the boilerplates.

Reply:
Moved away from this logic to a simpler condition based logic, the dynamic invocation needed some path calculation which was prone to bugs in future. Hence took the simpler route